Gem/Code/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@
 #
 #
 
+set(GAMELIFT_INTEGRATION OFF CACHE BOOL "Indicates if the game will be built with the GameLift integration features")
```

Comment:
I don't like this switch in the body of the project.
I would like to keep it cloud agnostic.
Please move it to a GameLift gem instead and export it to here. See sig-build for help with this.